src/components/input-field/partial-styles/outlined-style-overrides.scss

```diff
@@ -0,0 +1,63 @@
+.mdc-text-field {
+    transition: background-color 0.2s ease;
+
+    &:not(.mdc-text-field--disabled) {
+        background-color: $background-color-normal;
```

Comment:
Sounds like a scary time consuming task. Can we do a new separate issue for it?

Reply:
Doing it the way the docs tell you to do it is scarier than rocking your own?

Reply:
At the moment yes. Because I haven't read the docs yet… Maybe it's not that scary. But I prefer this not to block us from going further with this PR. What do you think?

Reply:
If you promise me to read the docs first next time…

Reply:
You know I'm dyslexic? Right? 😆 Yes I promise to read this too. It's just that I have a million to-dos which are now must-dos (bad excuse). But I'll read these soon.

Reply:
Yeah, but not doing it properly now is going to cause more work down the road. Will either of us have less to do then? 😕

Reply:
I guess we'll have to do some significant work on basically all elements when upgrading MDC, so I guess we can leave it… but it will be yet another thing that makes it even more difficult to upgrade… 😞

Reply:
I'm totally on your side! If I knew these things I promise I'd preform them in my work. We should do this soon. It also affects they way we will be handling dark/light themes. We should apply a good strategy there. Which is an additional reason for me to postpone this for future. I'm really looking forward to this design-oriented cycle and together going through what needs to be done in Lime Elements among other things.